src/components/form/fields/passwordField.jsx

```diff
@@ -170,18 +181,20 @@ export const passwordField = ({ label = 'password', canShowPassword = false, ...
                 strength: 0,
                 enabledRules: listEnabledRules(i18n, passwordPolicy),
                 isTouched: false,
-                isDirty: false
+                isDirty: false,
+                blacklist: [],
             }),
             unbind: (model, { value }) => ({ ...model, password: value }),
-            validate: ({ value, strength, isDirty }, ctx) => {
+            validate: ({ value, strength, isDirty, blacklist }, ctx) => {
                 if (!isDirty && !ctx.isSubmitted) return {};
 
                 const errors = [];
                 if (!value) {
                     errors.push(i18n('validation.required'));
                 }
                 else {
-                    if (strength < passwordPolicy.minStrength) {
+                    const includesBannedWords = blacklist.some(str => value.toLowerCase().includes(str));
```

Comment:
What about 'blacklist.includes(str.toLowerCase())' and always lower case the values when storing it into blacklist?
Edit: you've done it as I saw afterwards!

Reply:
Could indeed remove the redundant toLowerCase here.

However, your suggestion doesn't work: `str` here is an element in the array, hence the use of `some` (`exists` in Scala).